doc/overview/geostyler_overview.rst

```diff
@@ -0,0 +1,71 @@
+:Author: Jan Suleiman
+:Reviewer: Astrid Emde
+:Version: osgeolive14.0
+:License: Creative Commons Attribution 3.0 Unported (CC BY 3.0)
+
+
+@LOGO_geostyler@
+@OSGEO_KIND_geostyler@
+
+
+@NAME_geostyler@
+================================================================================
+
+@NAME_geostyler@ is an Open Source JavaScript library that enables users to style maps with the help of a Graphical User Interface.
+The goal of @NAME_geostyler@ is a universally applicable GUI that works with any given style format; may it be SLD, QML, OpenLayers, etc.
+
+@NAME_geostyler@ provides a set of UI Components for map styling. Just like a modular building block system all components can be stacked
+together to create a nice UI for your web applications. To simplify the setup, we also provide some high-level components
+(based on our building blocks) that already do the work for you. These include among many others Symbolizer Editors, RuleTables and
```

Comment:
Use some commas in that final sentence:
These include, among many others, Symbolizer blah blah blah
Or you could delete the "among many others" phrase because it's implied by the word "include".

Reply:
done in https://github.com/OSGeo/OSGeoLive-doc/pull/604/commits/79b892a57dc759bc650410399b965dbbf2bcdd69